lib/open_project/static/links.rb

```diff
@@ -98,6 +98,22 @@ def static_links
               href: 'https://www.openproject.org/docs/user-guide/',
               label: 'homescreen.links.user_guides'
             },
+            installation_guides: {
+              href: 'https://www.openproject.org/docs/installation-and-operations/installation/',
+              label: :label_installation_guides
+            },
+            packager_installation: {
+              href: 'https://www.openproject.org/docs/installation-and-operations/installation/packaged/',
+              label: 'Packaged installation'
+            },
+            docker_installation: {
+              href: 'https://www.openproject.org/docs/installation-and-operations/installation/docker/',
+              label: 'Docker installation'
+            },
+            manual_installation: {
+              href: 'https://www.openproject.org/docs/installation-and-operations/installation/manual/',
+              label: 'Docker installation'
```

Comment:
```suggestion
              label: 'Manual installation'
```

Well... only `href` is used for this one, but the label did not match

Reply:
Good catch, I wanted to add the label just in case it is used somewhere, and it should be correct.